modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-filtering-ebpf-rule_{context}"]
+= Filtering eBPF flow data with rules
+You can configure the `FlowCollector` to filter eBPF flows using rules as a method to control the flow of packets cached in eBPF's flows table based on certain configuration.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A few sample configurations are as follows:
++
+
+[%collapsible]
+.Filter Kubernetes service traffic to specific POD IP endpoint
+====
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:
+      flowFilter:
+        action: Accept  <1>
+        cidr: 172.210.150.1/24 <2>
+        protocol: SCTP
+        direction: Ingress
+        destPortRange: 80-100
+        peerIP: 10.10.10.10
+----
+<1> The required `action` parameter describes the action to be taken for the flow filter rule. Possible values are `Accept` and `Reject`. 
+<2> The required `cidr` parameter provides the IP address and CIDR mask for the flow filter rule, and supports IPv4 and IPv6 address formats. If you want to match against any IP address, you can use `0.0.0.0/0` or `::/0` for IPv4 and IPv6, respectively.
+====
++
+[%collapsible]
+.See flows to any addresses outside the cluster 
+====
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:
+      flowFilter:
+        action: Accept  
+        cidr: 0.0.0.0/0 <1>
+        protocol: TCP
+        direction: Egress
+        source_port: 100
+        peerIP: 192.168.127.12 <2>
```

Comment:
there is `enable` field to enable flow filtering feature without setting it to `true` feature won't work

Reply:
oh, ok. I asked about this on Slack, and I thought you said its not required but its useful for debugging:
> Does spec:
  agent.ebpf.metrics.enable need to be true for the filtering?
[4:45](https://redhat-internal.slack.com/archives/D031X53RGJZ/p1712954733783529)
I see its in Amogh's test


> mmahmoud
  4:46 PM
flow filtering has statistics so if user wanted to see if the flowfilter was matching or not then we need to enable metrics but its not need for filtering to work more for debugging , speaking of that we might have dashboard for flow filtering

Reply:
So based on this convo I think I just removed it instead of using a callout

Reply:
I added it back to both examples